docs/blog/2018-10-04-journey-to-the-content-mesh/index.md

```diff
@@ -0,0 +1,62 @@
+
+The CMS was born to make content sites _possible_. Now, it’s being reimagined to make them _incredible_.
+
+In the early 2000s, publishing on the web was difficult. The CMS emerged as a single application to store content, build sites, and deliver them to users.
+
+Over time, tools in each category got better and more specialized, while the quality of monolithic enterprise CMS applications like Adobe Experience Manager and Sitecore has stayed roughly the same.
+
+Today, a website team running an e-commerce site can store product inventory in [Shopify](https://www.shopify.com/), product listings in [Salsify](https://www.salsify.com/) and reviews in [Bazaarvoice](https://www.bazaarvoice.com/). A team running a paywalled content site can create stories in [Wordpress](https://wordpress.org/), store video in [JWPlayer](https://www.jwplayer.com/), user data in [Auth0](https://auth0.com/), and subscription data in [Recurly](https://recurly.com/).
+
+When website teams want to add search, they can turn to [Algolia](https://www.algolia.com/); for payments, [Stripe](http://stripe.com); for analytics, [Segment](https://segment.com/); for A/B testing, [Optimizely](https://www.optimizely.com/); for personalization, [Evergage](https://www.evergage.com/).
+
+Teams are adopting these services because each is tailored to a specific use-case, while the generic enterprise CMS alternative is optimized for none.
+
+This series is intended to explain the emergence of a content mesh -- the infrastructure layer for a decoupled website. The content mesh stitches together content systems in a modern development environment while optimizing website delivery for performance.
+
+The previous generation of enterprise CMS offered a set of mediocre components; a generic content editing experience glued to out-of-date development and deployment environments. The emerging strategy for content management selects best-of-breed solutions, tailored to specific use-cases like e-commerce or blogging; it provides a modern framework for rapid iteration and it generates lightning fast websites out of the box.
+
+Change is being spurred by the confluence of three revolutions in how we create and consume content:
+
+- _Content management._ Monolithic CMS applications are being replaced by modular, specialized content systems.
+
+- _Development techniques._ Modern UI frameworks like React are becoming essential to rich user experiences.
+
+- _Performance._ As mobile surpasses 50% of internet traffic, high-performance is becoming a must-have, not a nice-to-have.
+
+<figure>
+  <img alt="The modularization of the CMS" height="400" src="./modular-cms-architecture.png" />
+  <figcaption>
+   The modularization of the CMS
+  </figcaption>
+</figure>
+
+These technological changes have made modern web technology both _essential_ for creating fresh, novel, and engaging digital experiences, and _more accessible_ for the enterprise.
```

Comment:
How have they made modern web tech more accessible for enterprise companies?

Reply:
We'll get into that more in the rest of the series -- this is an overview piece.